### DISCUSSION TITLE
Linter Problem

### DISCUSSION THREAD

**Discussion by anresa** [neuefische/questions#31] [Web] [answered]
Hallo Zusammen,

ich werde aus diesem Linter Problem nicht schlau:

<img width="469" alt="Bildschirmfoto 2022-07-28 um 12 40 47" src="https://user-images.githubusercontent.com/106099037/181488099-2982751b-0c93-4b92-b9b1-1d0376e8abaf.png">

[Mein Repo dazu ](https://github.com/anresa/capstone-project)(auf dem aktuellen Stand da ich die Probleme schon länger habe.)

Habs bereits gegoogelt und da nur rausgefunden, dass es sich um einen SyntaxError handeln soll?! Also: "Es wurde eine bestimmte Sprachenstruktur erwartet, aber etwas anderes wurde übermittelt. Es kann sich dabei um einen einfachen Tippfehler handeln."

Ich habe an der Stelle/den Stellen aber keine Tippfehler und auch sonst kann ich keine Errors endtecken. Vielleicht fällt euch was ein. Danke schonmal!

**Comment by julialoeschel** [comment]
Hallo Anna, kannst du deinen Code mitschicken? 
es könnte tatsächlich eine fehlende Klammer sein oder ein `;`zuviel.

**Reply by anresa** [reply]
@julialoeschel Danke Julia, hab ich mit rein gepackt (mein Repo mit dem aktuellen Code).

**Reply by mbosselmann** [reply]
Hallo Anna,
ich finde auf GitHub keinen Branch mit deinem Problem. Hast du dein Problem bereits gelöst? Oder ist dort der neueste Stand noch nicht  vorhanden? 🤔

**Comment by pixelass** [comment, marked as answer]
Das Template ist eigentlich für styled-components vorbereitet, bei dir taucht der Fehler in der `App.css` Datei auf.

Wenn du dennoch mit `.css` Dateien arbeiten willst schalte stylelint für das CSS aus:

In https://github.com/anresa/capstone-project/blob/main/package.json#L14

Vorher: 

```json
{
  "stylelint": "npx stylelint src/{**/*,*}.{css,js,jsx}",
}
```

Nachher: 

```json
{
  "stylelint": "npx stylelint 'src/{**/*,*}.{js,jsx}'",
}
```
